prow.sh

```diff
@@ -230,6 +230,7 @@ configvar CSI_PROW_E2E_IMPORT_PATH "k8s.io/kubernetes" "E2E package"
 
 # Local path for e2e tests. Set to "none" to disable.
 configvar CSI_PROW_SIDECAR_E2E_IMPORT_PATH "none" "CSI Sidecar E2E package"
+configvar VOLUME_MODE_CONVERSION_TESTS "false" "Enable volume mode conversion feature flag. Used in sidecar E2E tests."
```

Comment:
This doesn't follow the `CSI_PROW_*` naming convention.

Just saying - I still think this entire change to csi-release-tools should better get moved into the one repo where it actually would have an effect.

Reply:
Okay, sounds good, I'll make the changes to external-provisioners `.prow.sh`. Keeping this PR open for now